tests/src/system/basic/Swift3WhiskObjectTests.scala

```diff
@@ -80,14 +83,17 @@ class Swift3WhiskObjectTests
             val run = wsk.action.invoke(actionName, Map("triggerName" -> triggerName.toJson))
             withActivation(wsk.activation, run, initialWait = 5 seconds, totalWait = 60 seconds) {
                 activation =>
-                    activation.fields("logs").toString should include(s"Tigger Name: $triggerName")
+                    // should be successful
+                    activation.fields("response").asJsObject.fields("success").toString should be("true")
+
+                    // should have a field named "activationId" which is the date action's activationId
+                    activation.fields("response").asJsObject.fields("result").asJsObject.fields("activationId") should not be (null)
```

Comment:
how about doing `....toString.nonEmpty should be(true)` instead of checking null. In fact since the length of the activation id is known to be 32.